samples/nrf9160/icalendar_parser/cert/GlobalSign-Root-CA-R2

```diff
@@ -0,0 +1,90 @@
+/*
+   Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            04:00:00:00:00:01:0f:86:26:e6:0d
+        Signature Algorithm: sha1WithRSAEncryption
+        Issuer: OU = GlobalSign Root CA - R2, O = GlobalSign, CN = GlobalSign
+        Validity
+            Not Before: Dec 15 08:00:00 2006 GMT
+            Not After : Dec 15 08:00:00 2021 GMT
```

Comment:
I would be a bit careful in including root certificates. This one expires next year December so if you forget to update the sample it will suddenly stop working. I would rather describe how they can get the root certificate. I do however see that it can be a convenience to include it.

Reply:
I agree that it will expire soon. Since the Google root cert is already described in the README.rst, I will remove the root cert and make it as a template.

Reply:
Awesome 👍